website/static/js/pages/project-dashboard-page.js

```diff
@@ -34,12 +34,14 @@ $('body').on('nodeLoad', function(event, data) {
 });
 
 // Initialize comment pane w/ it's viewmodel
-var $comments = $('#comments');
+var $comments = $('.comments');
 if ($comments.length) {
     var userName = window.contextVars.currentUser.name;
     var canComment = window.contextVars.currentUser.canComment;
     var hasChildren = window.contextVars.node.hasChildren;
-    Comment.init('#commentPane', userName, canComment, hasChildren);
+    var node_id = window.contextVars.node.id;
+    Comment.init('.commentPane', 'node', node_id, 'pane', userName, canComment, hasChildren);
```

Comment:
There are multiple commentPane s in the whole project, and so I think it would better be a class than an Id